admin/src/components/Injector/index.js

```diff
@@ -19,7 +19,7 @@ const Injector = () => {
     isLoading,
     isSupportedType,
     url,
-  } = usePreviewUrl( uid, initialData, isDraft, isCreatingEntry );
+  } = usePreviewUrl( uid, modifiedData, isDraft, isCreatingEntry );
```

Comment:
@Dupflo Can you remove the `initialData` from the destructuring above? It looks like it is no longer used here.

Reply:
@mattmilburn done !